ci-operator/config/openstack-k8s-operators/test-operator/openstack-k8s-operators-test-operator-18.0.0-proposed.yaml

```diff
@@ -0,0 +1,93 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  openstack-operator-ci-tempest:
```

Comment:
This is only needed if you plan to add  a job that runs tempest step at the end. Otherwise, you can remove L6 to L9.
I think that we can improve our tempest step to actually run test-operator, instead of: https://steps.ci.openshift.org/reference/openstack-k8s-operators-tempest

But we can do that in a follow-up?

Reply:
since this is the operator to run tempest with, I am not sure if it makes sense. I'll remove it for now